Orange/canvas/application/addons.py

```diff
@@ -565,6 +557,7 @@ def __on_installer_error(self, command, pkg, retcode, output):
 
     def __on_installer_finished(self):
         message_information(
+            "Changes successfully applied in <i>{}</i>.<br>".format(USER_SITE) +
```

Comment:
This now prints that the changes have been made in the local user dir even if `user_install` is False and they were not.